### PR TITLE
fix: release

### PR DESCRIPTION
**Pull request by Dav-14**

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -9,34 +9,8 @@ permissions:
...
-          if (( $(echo ${#hasChanged}) != 0 )); then
-            git status
-            echo "There are changes in the repository"
-            exit 1
-          fi
-  
   release:
     name: Release
-    needs:
-      - tests
```
Comment (Dav-14) [inline .github/workflows/release.yml:39]: Not needed anymore, it it executed within the +pre-commit

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -53,6 +27,18 @@ jobs:
...
+      - name: Has filesystem changed ?
+        id: changed-files
+        shell: bash
+        run: |
+          hasChanged=$(git status --porcelain) 
+          if (( $(echo ${#hasChanged}) != 0 )); then
+            git status
+            echo "There are changes in the repository"
+            exit 1
+          fi
```
Comment (Dav-14) [inline .github/workflows/release.yml:41]: Pre-commit, Test filesystem, then release

Comment (Dav-14) [reply .github/workflows/release.yml:41]: I would have loved to have something directly in the release target but it's harder

File: Earthfile [hunk 3]
```diff
@@ -27,23 +27,20 @@ package:
...
+  BUILD +package
 
 release:
   FROM core+builder-image
   GIT CLONE --branch=v1.6.1 https://github.com/helm/chart-releaser /src/chart-releaser
   WORKDIR /src/chart-releaser
   DO core+GO_INSTALL --package ./...
   COPY ./cr.yaml .
-  COPY (+ci/*) /build
+  COPY (+package/*) /build
```
Comment (Dav-14) [inline Earthfile:43]: AS pre-commit must be done before we and +ci does not export any artifact anymore

File: .github/workflows/pull_request.yml [hunk 4]
```diff
@@ -73,7 +73,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Tests
-        run: earthly +ci
+        run: earthly +pre-commit
```
Comment (flemzord) [inline .github/workflows/pull_request.yml:76]: Why ?

File: .github/workflows/release.yml [hunk 5]
```diff
@@ -9,34 +9,8 @@ permissions:
   packages: write
 
 jobs:
-  tests:
```
Comment (flemzord) [inline .github/workflows/release.yml:12]: No tests need to be run before release

File: Earthfile [hunk 6]
```diff
@@ -27,23 +27,20 @@ package:
   COPY (./charts/*+package/*) /build/
   SAVE ARTIFACT /build AS LOCAL ./
 
-ci:
```
Comment (flemzord) [inline Earthfile:30]: Need for CI

File: Earthfile [hunk 7]
```diff
@@ -27,23 +27,20 @@ package:
...
-ci:  
-  BUILD +pre-commit
-  BUILD +tests # This target could depend on updated dependencies with the env variable NO_UPDATE
-  BUILD +package
-
 pre-commit:
   BUILD --pass-args +validate
   BUILD +template --TEMPLATE_FILE=readme.tpl --OUTPUT_FILE=README.md
   BUILD +template --TEMPLATE_FILE=contributing.tpl --OUTPUT_FILE=CONTRIBUTING.md
+  BUILD +tests # This target could depend on updated dependencies with the env variable NO_UPDATE
```
Comment (flemzord) [inline Earthfile:34]: To slow for local pre-commit

**Review by flemzord — CHANGES_REQUESTED** [review]

File: .github/workflows/release.yml (hunk 5 above)
Comment (Dav-14) [reply .github/workflows/release.yml:12]: There is actualy test before release, i can remove the tests job and keep the +release target with the +package sources

File: Earthfile (hunk 6 above)
Comment (Dav-14) [reply Earthfile:30]: Is the pre-commit not sufficient ?

File: .github/workflows/pull_request.yml (hunk 4 above)
Comment (Dav-14) [reply .github/workflows/pull_request.yml:76]: doing the same now

**Review by flemzord — APPROVED** [review]

**Pull request merged**